.scalafmt.conf

```diff
@@ -1,4 +1,23 @@
-maxColumn = 120
-version = 3.5.9
-runner.dialect = scala212source3
-preset = IntelliJ
\ No newline at end of file
+runner.dialect = scala213source3
+
+version                    = "3.7.14"
+maxColumn                  = 120
+align.preset               = more
+rewrite.rules              = [AsciiSortImports]
+spaces.inImportCurlyBraces = false
+
+danglingParentheses.defnSite             = true
+danglingParentheses.ctrlSite             = false
+danglingParentheses.callSite             = true
+align.closeParenSite                     = true
+newlines.implicitParamListModifierPrefer = before
+
+trailingCommas = keep
+docstrings.style = keep
```

Comment:
I'm wondering if it would be a good idea to add some comments around these customization to be easier to figure out why we add them?